Simple patch to fix the section number in the .TH line for the man
page.  This is suitable for upstream submission.

--- a/man/man6/common_test.6	2020-02-25 13:34:44.318321275 +0000
+++ b/man/man6/common_test.6	2020-02-25 13:34:15.249314188 +0000
@@ -1,4 +1,4 @@
-.TH common_test 3 "common_test 1.18.1" "Ericsson AB" "Erlang Module Definition"
+.TH common_test 7 "common_test 1.18.1" "Ericsson AB" "Erlang Module Definition"
 .SH NAME
 common_test \- A framework for automated testing of any target nodes.
 .SH DESCRIPTION
